examples/custom-image-extension/package.json

```diff
@@ -3,7 +3,7 @@
   "displayName": "Foxglove Example Custom Image Extension Panel",
   "description": "",
   "publisher": "Foxglove",
-  "version": "1.0.2",
+  "version": "1.0.3",
```

Comment:
Doesn’t really seem like we need to be bumping these version numbers since it is an example and the package isn’t published anywhere